lib/src/Uncertainty/StatTests/LinearModelTest.cxx

```diff
@@ -85,7 +85,7 @@ TestResult LinearModelTest::LinearModelFisher(const Sample & firstSample,
 TestResult LinearModelTest::LinearModelFisher(const Sample & firstSample,
     const Sample & secondSample,
     const LinearModel & linearModel,
-    const Scalar level)
+    const Scalar )
```

Comment:
level was probably missing in the call to `LinearModelFisher(firstSample, secondSample, linearModelResult)`

Reply:
done